src/commands/sourcemaps/upload.ts

```diff
@@ -114,7 +114,7 @@ export class UploadCommand extends Command {
   }
 
   private getMatchingSourcemapFiles(): Payload[] {
-    const sourcemapFiles = glob.sync(buildPath(this.basePath!, '**/*.js.map'))
+    const sourcemapFiles = glob.sync(buildPath(this.basePath!, '**/*js.map'))
```

Comment:
Looks like we don't have matching tests... should we add some?

Reply:
I added a test case in `upload.test.ts` that covers it. 👍